src/WorkBC.Web/Controllers/IndustryProfilesController.cs

```diff
@@ -61,25 +61,23 @@ public async Task<ActionResult<IEnumerable<IndustryProfileModel>>> GetSavedIndus
             //    ? await GetIndustryIdsAndJobCounts()
             //    : new Dictionary<int, IndustryProfileModel>();
 
-            var query = from p in _enterpriseContext.IndustryProfiles
-                        where savedIndustryProfilesDict.Keys.Contains(p.IndustryProfileId)
-                        orderby p.PageTitle
-                        select new IndustryProfileModel
-                        {
-                            Id = savedIndustryProfilesDict[p.IndustryProfileId],
-                            Title = p.PageTitle,
-                            //Count = new Random().Next(2000)
-
-                            // todo: don't delete this until we confirm that we are not adding job search counts and links to saved industry profiles
-                            //Count = jobSearchInfo.ContainsKey(p.IndustryProfileId)
-                            //     ? jobSearchInfo[p.IndustryProfileId].Count
-                            //     : 0,
-                            //IndustryIds = jobSearchInfo.ContainsKey(p.IndustryProfileId)
-                            //     ? jobSearchInfo[p.IndustryProfileId].IndustryIds
-                            //     : ""
-                        };
-
-            return Ok(await query.ToListAsync());
+            var industryProfiles = await _ssotApi.GetIndustryProfiles(savedIndustryProfilesDict.Keys.ToList());
+            
+            var results = new List<IndustryProfileModel>();
+            
+            if (industryProfiles.Count > 0)
```

Comment:
Don't need this if condition because the loop will not run if the list is empty.